install_pim/docker/installation_docker.rst

```diff
@@ -233,80 +95,62 @@ Now, you can initialize Akeneo by running:
 
 .. code-block:: bash
 
-   $ bin/docker/pim-dependencies.sh
-   $ bin/docker/pim-initialize.sh
-
-Those two bash scripts are just helpers placed in the PIM, in the folder ``bin/docker``. They execute the following commands (you could do so too if you prefer):
-
-- ``pim-dependencies.sh``
-
-.. code-block:: bash
-
-   $ docker-compose exec fpm composer update
+   $ export APP_ENV=prod
+   $ docker-compose run --rm php php -d memory_limit=4G /usr/local/bin/composer install
    $ docker-compose run --rm node yarn install
+   $ docker-compose run --rm php php bin/console pim:installer:assets --symlink --clean
```

Comment:
We should document the `make` commands instead of docker_compose, because this is what we will us ourselves.